packages/ic-asset-certification/tests/common/mod.rs

```diff
@@ -0,0 +1,55 @@
+use ic_asset_certification::ASSET_CHUNK_SIZE;
+use ic_response_verification_test_utils::hash;
+use rand_chacha::{rand_core::{RngCore, SeedableRng}, ChaCha20Rng};
+
+pub fn asset_chunk<'a>(asset_body: &'a [u8], chunk_number: usize) -> &'a [u8] {
+    let start = chunk_number * ASSET_CHUNK_SIZE;
+    let end = start + ASSET_CHUNK_SIZE;
+    &asset_body[start..end.min(asset_body.len())]
+}
+
+pub fn asset_body(asset_name: &str, asset_size: usize) -> Vec<u8> {
+    let mut rng = ChaCha20Rng::from_seed(hash(asset_name));
+    let mut body = vec![0u8; asset_size];
+    rng.fill_bytes(&mut body);
+
+    body
+}
+
+#[macro_export]
+macro_rules! assert_contains {
```

Comment:
Writing macros for this is a bit of an experiment, but it seems to work well. It makes the tests cleaner in my opinion and gives reasonable error messages for the correct line number (meaning the line number of where the macro is called from, rather than the line number of the implementation) when it fails.